src/lib.rs

```diff
@@ -808,7 +815,12 @@ impl Config {
                 {
                     cmd.env("MAKEFLAGS", makeflags);
                 }
-                _ => {}
+                _ => {
+                    if !cmake_has_parallel && num_jobs > 1 {
+                        println!("cargo:warning=The parallel flag was not passed to CMake since the detected version is below 3.12!");
```

Comment:
If the `-j` flag is passed to `make` then I don't think that this warning should be printed since parallel builds do happen. This should probably only be reserved for builds where parallelism isn't specified.

Reply:
I also think it would be helpful to print out the version detected here as well.